src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -2039,9 +2045,11 @@ void Interpolate::initSupportedPrimitiveDescriptors() {
         outputPrecision = fusedWith[fusedWith.size() - 1]->getOriginalOutputPrecisionAtPort(DATA_ID);
     }
 
+#if defined(OPENVINO_ARCH_X86_64)
```

Comment:
I woul prefer to keep this condition for risc-v. So we should skip it only for OV_CPU_WITH_ACL case

Reply:
Fixed